src/modules/ekf2/EKF/aid_sources/drag/drag_fusion.cpp

```diff
@@ -57,6 +57,10 @@ void Ekf::controlDragFusion(const imuSample &imu_delayed)
 
 		if (_drag_buffer->pop_first_older_than(imu_delayed.time_us, &drag_sample)) {
 			fuseDrag(drag_sample);
+
+			if (!_aid_src_drag.fused && !_control_status.flags.fuse_aspd && !_control_status.flags.fuse_beta) {
```

Comment:
I don't think we would like to reset as soon as a measurement spike is rejected. Also, `fuseDrag` returns directly if the mcoef and bcoef are not set (https://github.com/PX4/PX4-Autopilot/pull/23277/files#diff-5cf72a0fdf96e3bc1019002679b271a5ccf1b96106efa83240576205ec5893f8L79); in that case we also don't want to reset.
I think you would need to do a slightly more complicated start/stop logic if you want to call a reset in here